docs/config.md

```diff
@@ -55,6 +55,35 @@ for more details.
 | `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | * | source |
 | `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime) | * | source |
 
+## Logs
+
+Logs from ASP.NET Core apps could be captured using two options. First option is
+to use HostingStartup and the second option is to rely on CLR Profiler
+Instrumentation. Recommended option for ASP.NET Core is to use HostingStartup.
+For other .NET Core apps logs could be collected only through .NET CLR Profiler
+Instrumentation.
+
+### Enable Logs using HostingStartup
+
+| Environment variable | Required value |
+|-|-|
+| `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` | `OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper` |
+
+### Enable Logs using .NET CLR Profiler
+
+Please follow [instructions](config.md#net-clr-profiler) here to collect logs
+using .NET CLR Profiler.
```

Comment:
I see this PR as adding "bytecode instrumentation" for `ILogger` 

Maybe we could try to describe it as such in a `Instrumented logging libraries and frameworks` section?

Reply:
There are no additional instrumentation options like traces or metrics, hence placed in a different section. Even if we add `Instrumented logging libraries and frameworks`, there will be only `AspNet` option. 

If I follow you correct, `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` should get to source instrumentation and  .NET CLR Profiler part will move to `Instrumented logging frameworks` section. Is that correct?

Reply:
I think it may be easier if we merge this and I will try to create a PR 😋 At least I think it will be easier and faster for me 😉